dashboard/public/l10n/locales.ini

```diff
@@ -776,7 +776,7 @@ ClassroomUpdated=कक्षा {{name}} को सफलतापूर्व
 ClassroomCreated=कक्षा {{name}} सफलतापूर्वक बनाया गया है!
 ClassroomDeleted=कक्षा {{name}} को सफलतापूर्वक हटा दिया गया है!
 DoYouWantDeleteClassroom=क्या आप कक्षा {{classroom}} को हटाना चाहते हैं?
-ShortForBytes=ब
```

Comment:
This should not be changed. The data is in Bytes not Bits.

Reply:
Thanks for the review. Although shouldn't this follow the common trend of using बी for Bytes? For example in ShortForKilobytes=केबी and ShortForMegabytes=एमबी.

Reply:
If you use बी for Byte then it could cause confusion between Bit and Byte.
@ashish0910 what do you think?

Reply:
I think ब makes more sense .

Reply:
Fixed in latest commit.